x-pack/plugins/timelines/server/search_strategy/timeline/index.ts

```diff
@@ -93,6 +95,7 @@ const timelineSearchStrategy = <T extends TimelineFactoryQueryTypes>({
   options: ISearchOptions;
   deps: SearchStrategyDependencies;
   queryFactory: TimelineFactory<T>;
+  logger: Logger;
```

Comment:
I think we can remove this changes. Because do not use it any more.

Reply:
replied in other comment.